VERSION

```diff
@@ -1,4 +1,4 @@
-VERSION_MAJOR = 2
-VERSION_MINOR = 0
-VERSION_PATCH = 0
-VERSION_SUFFIX = -dev
+VERSION_MAJOR=2
+VERSION_MINOR=0
+VERSION_PATCH=0
```

Comment:
Just out of curiosity, any particular reason to call this one `PATCH` and not `MICRO`?

Reply:
We follow [SemVer](https://semver.org/) :)

Reply:
well perfect!